scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -12,10 +12,25 @@ import scala.meta.internal.{semanticdb => s}
 import scala.meta.internal.semanticdb.Scala._
 import scala.meta.internal.semanticdb.Scala.{Descriptor => d}
 import scala.meta.internal.semanticdb.Scala.{Names => n}
+import scala.meta.internal.semanticdb.{SymbolInformation, TypeRef, TypeSignature}
```

Comment:
The idiomatic way of referring to these data structures is `import scala.meta.internal.{semanticdb => s}` and then `s.SymbolInformation`, `s.TypeRef`, etc.